dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EntryResource.java

```diff
@@ -421,6 +433,101 @@ public Entry setDiscourseTopic(@ApiParam(hidden = true) @Parameter(hidden = true
         return createAndSetDiscourseTopic(id);
     }
 
+    @Path("/updateLanguageVersions")
+    @RolesAllowed("admin")
+    @Deprecated
+    @UnitOfWork
+    @POST
+    @Operation(operationId = "updateLanguageVersions", description = "Update language versions", security = @SecurityRequirement(name = JWT_SECURITY_DEFINITION_NAME))
```

Comment:
If you end up going down this route (undecided myself), you also do everything "past" a particular ID as well if you don't want to mess with dates/times

Reply:
I'm now thinking it's worth keeping this endpoint around, just in case. For example, let's say we have a bug parsing some WDL 1.1 files and so we fail to determine the version when the user updated the workflow. We then fix the bug and we want to rerun the parsing for workflows where it failed.

We could either rerun the whole thing, or we could a parameter that takes a list of ids -- note that in this scenario, the ids could be interspersed -- the bug could even crop up on entry with id 1, if the user recently updated a version on that entry.

Proposal:

* Keep the endpoint around
* Annotate it with `@Hidden` instead of `@Deprecated`
* Add additional query params, e.g., language type, list of ids, last modified date, if the need arises, because they would only be useful with additional code changes any way, e.g., the hypothetical WDL bug fix.

Reply:
`@Hidden` was a bad idea -- I thought it just wouldn't show up in the UI, but it also doesn't generate client code.

Reply:
(no strong feelings, just that we describe when we use this and/or if/when we remove it)

Reply:
I just ended up removing `@Deprecated`, so the endpoint will always be there, for reasons explained above -- hopefully we'll never need to run it again, but as I said, I could see bugs cropping up in our language processing where we might need to.